### DISCUSSION TITLE
Allow using X509Certificate instead of Pfx

### DISCUSSION THREAD

**Discussion by KieranFoot** [jchristn/HttpServerLite#32] [Q&A]
In my attempted use of this library, I would need to pass an X509Certificate which I resolve from the local computer.
This is because it will be installed and trusted ahead of time.

**Comment by jchristn** [comment]
Were you aware of this constructor?
```
public Webserver(string hostname, int port, Func<HttpContext, Task> defaultRoute, X509Certificate2 sslCertificate = null)
```
